transcendence-app/src/shared/db/base.repository.ts

```diff
@@ -0,0 +1,18 @@
+import { ID } from './models';
+
+export type DataResponseWrapper<T> = {
```

Comment:
I have mixed feelings about this. Maybe we could log errors next to the db driver and returning `null`, that is a good idea :+1: 

But, what about controlling, for example, if you succeeded updating data into the db? you may try to write data into a username that does not exist. That scenario does not throw any errors and just returns a 0 value for entry which could be useful to know if we succeeded.

Reply:
We could add [`RETURNING *`](https://www.postgresql.org/docs/14/dml-returning.html) at the end of I`NSERT`, `UPDATE`, and `DELETE` queries.

Reply:
Ok! I see your point. So I'll update the repository to always return data on success. We'll then return `null` values when recieving not successful queries responses and will throw errors in case we receive any. Thanks!

Reply:
Updated! I left, as said in other comments the task of handling errors for the logging task